cartoframes/io/managers/context_manager.py

```diff
@@ -394,18 +396,18 @@ def _not_reserved(column):
 
 
 def _create_table_from_columns_query(table_name, columns):
-    columns = ['{name} {type}'.format(name=c.dbname, type=c.dbtype) for c in columns]
+    columns = ['{name} {type}'.format(name=double_quote(c.dbname), type=c.dbtype) for c in columns]
     return 'CREATE TABLE {table_name} ({columns})'.format(
         table_name=table_name,
-        columns=', '.join([double_quote(c) for c in columns]))
+        columns=','.join(columns))
 
 
 def _create_table_from_query_query(table_name, query):
     return 'CREATE TABLE {table_name} AS ({query})'.format(table_name=table_name, query=query)
 
 
 def _cartodbfy_query(table_name, schema):
-    return "SELECT CDB_CartodbfyTable('{schema}', '{table_name}')".format(
+    return 'SELECT CDB_CartodbfyTable(\'{schema}\', \'{table_name}\')'.format(
```

Comment:
Is this change needed? The query was more legible before

Reply:
I prefer using ' ' and then skip to be consistent with other parts of the code